sa/sa.go

```diff
@@ -1376,9 +1517,8 @@ func AuthzMapToPB(m map[string]*core.Authorization) (*sapb.Authorizations, error
 	return resp, nil
 }
 
-// NewAuthorizations2 adds a set of new style authorizations to the database and returns
-// either the IDs of the authorizations or an error. It will only process corepb.Authorization
-// objects if the V2 field is set. This method is intended to deprecate AddPendingAuthorizations
+// NewAuthorizations2 adds a set of new style authorizations to the database and
+// returns either the IDs of the authorizations or an error.
```

Comment:
We should file a follow-up to convert NewAuthorization2 to the MultiInserter, since it's used by ra.NewAuthorization as well.

Reply:
Yeah, I was of two minds on this. I could definitely convert NewAuthorization2 to use the multi-inserter right away. But I was thinking of leaving it as-is for now, and essentially using the feature flag to do controlled testing of the multi-inserter too. Only switch over other methods to use it after it's been proven in staging, basically.

Reply:
I agree, a follow-up would be best.

Reply:
Filed https://github.com/letsencrypt/boulder/issues/5624